crates/fhir-sdk/src/client/r5/paging.rs

```diff
@@ -150,14 +150,9 @@ impl Stream for Paged {
 	}
 }
 
-/// Find the URL of the next page of the results returned in the Bundle.
-fn find_next_page_url(bundle: &Bundle) -> Option<&String> {
-	bundle
-		.link
-		.iter()
-		.flatten()
-		.find(|link| link.relation == LinkRelationTypes::Next)
-		.map(|link| &link.url)
+/// Find the URL of the page of the results returned in the Bundle.
+pub(crate) fn find_page_url(bundle: &Bundle, relation_type: LinkRelationTypes) -> Option<&String> {
```

Comment:
Maybe move this to a upper level helpers module instead, `client::misc.rs`?

Reply:
Every version has different types here, so it is not quite as easy as making one function for all versions in a helper module sadly. But I have implemented the trait way on main already, so that is solved. I plan on moving it to fhir-model though out of the trait and using the trait for other things possibly.